helm/cluster/files/etc/containerd/config.toml

```diff
@@ -31,6 +31,9 @@ sandbox_image = "{{ include "cluster.image.registry" $ }}/{{ $.Values.providerIn
         {{- if and $.Values.global.components.containerd.localRegistryCache.enabled (has $host $.Values.global.components.containerd.localRegistryCache.mirroredRegistries) -}}
         "http://127.0.0.1:{{ $.Values.global.components.containerd.localRegistryCache.port }}",
         {{- end -}}
+        {{- if and $.Values.global.components.containerd.managementClusterRegistryCache.enabled (has $host $.Values.global.components.containerd.managementClusterRegistryCache.mirroredRegistries) -}}
+        "https://zot.{{ $.Values.global.managementCluster }}.{{ $.Values.global.connectivity.baseDomain }}",
+        {{- end -}}
         {{- range $value := $config -}}
         "https://{{$value.endpoint}}",
```

Comment:
just for my understanding, will this not add always to the last endpoint a ",". How can this work in the config?

Example:
CONFIG.TOML
...
zot.something.com,
myendpoint.something.com,

I assume the "," is only valid if another mirror registry is added, right?

Reply:
Trailing commas (ending an array with a comma, without an element behind it) are allowed in TOML, see here https://toml.io/en/v1.0.0#array.

> Arrays can span multiple lines. A terminating comma (also called a trailing comma) is permitted after the last value of the array.